buku

```diff
@@ -232,7 +695,7 @@ def getPageResp(url, fullurl=False):
     urlconn.request("GET", url, None, {
         "Accept-encoding": "gzip",
     })
-    return (urlconn, urlconn.getresponse())
+    return urlconn, urlconn.getresponse()
```

Comment:
Please retain the brackets. It's a [valid practice](http://stackoverflow.com/questions/9752958/how-can-i-return-two-values-from-a-function-in-python).